frontend/packages/console-app/src/plugin.tsx

```diff
@@ -59,7 +63,10 @@ type ConsumedExtensions =
   | DashboardsOverviewHealthPrometheusSubsystem
   | DashboardsOverviewInventoryItem
   | DashboardsOverviewHealthOperator<ClusterOperator>
-  | ReduxReducer;
+  | ReduxReducer
```

Comment:
I would prefer it if we do not pass it via plugins. We already have Nodes, Volume Snapshot, and Volume Snapshot Class on the resource page. Let's not mix these two. Or let's do it the other way and move the aforementioned resources here.
Let's be consistent.

Reply:
I prefer using the extension points, I find it easier to maintain.